src/uncrustify.cpp

```diff
@@ -38,10 +38,55 @@
 #include <vector>
 #include <deque>
 
+
+static const char *pcf_names[] =
```

Comment:
I missed that, was only needed a '#define DEFINE_PCF_NAMES' before import 'uncrustify_types.h', because the Eclipse IDE was not finding it any anywhere.